backend/app/main.py

```
@@ -156,17 +156,20 @@ async def chat_with_counselor(chat_message: ChatMessage):
-            model="gpt-4",
-            messages=[
-                {"role": "system", "content": system_prompt},
-                {"role": "user", "content": message}
-            ],
-            max_tokens=1000,
-            temperature=0.7
-        )
-        
-        ai_response = response.choices[0].message.content
+        # Mock response for testing when OpenAI API is not available
+        if not os.getenv("OPENAI_API_KEY") or "insufficient_quota" in str(e) if 'e' in locals() else False:
```

Ruff (0.12.2)

Undefined name `e`

(F821)